net-analyzer/bro/bro-2.4.1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{3,4,5} )
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="The Bro Network Security Monitor"
+HOMEPAGE="https://www.bro.org"
+SRC_URI="https://www.bro.org/downloads/release/${PF}.tar.gz"
```

Comment:
This should probably be ``${P}.tar.gz`` since ``${PF}`` will include any potential future revision number.